### DISCUSSION TITLE
Metronome and change record count-in

### DISCUSSION THREAD

**Discussion by JonasAlmfeldt** [TheKikGen/MPCLiveXplore-libs#22] [Q&A]
Hi,

Awsome work with iamforce2!!
Thanks a lot!

I wonder how/where can I find the metronome settings in FORCE mode?

I would like to increase the record count-in. feel its too short by default.

Additional question: will you upgrade to MPC version 2.12.3.x soon?

Kind regards,
Jonas

**Comment by Pitrakos** [comment]
Hi

If you have a launchpad connected like the Novation launchpad Mini Mk3, you have to press STOP/SOLO/MUTE button plus launch scene 5 button plus leave press button column 5 file 8 for a while and the metronome window will pop up. It´s sounds a little bit messy but is the way for show that metronome window. 
With not launchpad controller from the MPC ONE itself I never discovered how to open that metronome window.

May this help you.

Regards
